azure-configurationservice/azure/configuration/_generated/_azure_configuration_client_imp.py

```diff
@@ -0,0 +1,38 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+
+from msrest.service_client import SDKClient
+from msrest import Serializer, Deserializer
+
+from ._configuration import AzureConfigurationClientImpConfiguration
+from .operations import AzureConfigurationClientImpOperationsMixin
+from . import models
+
+
+class AzureConfigurationClientImp(AzureConfigurationClientImpOperationsMixin, SDKClient):
+    """Represents an azconfig client
```

Comment:
Do we care about this type of comments to follow the actual name of the service? like i.e. App Configuration Client?

Reply:
Agree. By changing the swagger file I can change the auto-generated comments